.drone.yml

```diff
@@ -9,21 +9,27 @@ clone:
     tags: true
 
 pipeline:
+  pre-build:
+    image: webhippie/nodejs:latest
+    pull: true
+    commands:
+      - npm install
+      - make stylesheets-check
+    when:
+      event: [ push, tag, pull_request ]
   build:
     image: webhippie/golang:edge
     pull: true
     environment:
       TAGS: bindata sqlite
       GOPATH: /srv/app
     commands:
-      - apk -U add nodejs nodejs-npm
-      - npm install
       - make clean
+      - make minify
```

Comment:
`minify` can be moved after `stylesheets-check` to prevent unnecessary build step execution

Reply:
It has to be runned before `make generate`.